src/import/chips/p9/procedures/utils/stopreg/p9_stop_api.C

```diff
@@ -865,17 +874,11 @@ StopReturnCode_t p9_stop_save_scom( void* const   i_pImage,
             break;
         }
 
-        uint32_t nopInst = getOriInstruction( 0, 0, 0 );
-
-        ScomEntry_t* pEntryLocation = NULL;
-        ScomEntry_t* pNopLocation = NULL;
-        ScomEntry_t* pTableEndLocationtable = NULL;
-        uint32_t swizzleAddr = SWIZZLE_4_BYTE(i_scomAddress);
-        uint64_t swizzleData = SWIZZLE_8_BYTE(i_scomData);
-        uint32_t swizzleAttn = SWIZZLE_4_BYTE(ATTN_OPCODE);
-        uint32_t swizzleEntry = SWIZZLE_4_BYTE(SCOM_ENTRY_START);
-        uint32_t index = 0;
-        uint32_t swizzleBlr = SWIZZLE_4_BYTE(BLR_INST);
+	nopInst = getOriInstruction( 0, 0, 0 );
```

Comment:
From Vance - If you want identical behavior, I think you still want to NULL your 3 ScomEntry_t pointers here. There's a possibility a pointer gets set one time around the loop and not changed the next time around.